.travis.yml

```diff
@@ -1,11 +1,18 @@
+sudo: false
 language: python
 python:
   - "2.7"
 
-before_install:
-  - "sudo apt-get build-dep python-scipy"
+addons:
+  apt:
+    packages:
+      # sciypy dependency
+      - gfortran
+      - libopenblas-dev
+      - liblapack-dev
 
-install: travis_wait 60 "bin/bootstrap.sh $VIRTUAL_ENV"
```

Comment:
In the past, Travis timed out without "travis_wait" - is it no longer necessary>